src/index.ts

```diff
@@ -103,6 +105,16 @@ function validateConfig(o: JsObject) {
     if ('loadStructure' in o && o.loadStructure !== undefined && !isFunction(o.loadStructure)) {
         throw Error('"loadStructure" should be a function in chemiscope config');
     }
+
+    if ('maxWidgets' in o) {
+        if (typeof o.maxWidgets !== 'number' || o.maxWidgets === null) {
```

Comment:
why the null check here? `typeof` should already cover this, right? There is a null check when the expected type is `"object"`, since `typeof null === "object"`.